routes/schedule_route.py

```
@@ -9,18 +9,23 @@
 from schemas.schedule_schema import ScheduleResponseSchema
 from utils.database import get_db
+from utils.jwt_utils import verify_token
 
 schedule_router = APIRouter()
 
-@schedule_router.get("/{user_id}", response_model=List[ScheduleResponseSchema])
-def get_schedule(user_id: int, db: Session = Depends(get_db)):
+@schedule_router.get("/", response_model=List[ScheduleResponseSchema])
+def get_schedule(
+    token_user_id: str = Depends(verify_token),  # JWT → user_id
+    db: Session = Depends(get_db),
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)